apps/helloworld/quarkus/poller.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+while true
+do 
+  curl $(minikube -p 9steps ip):$(kubectl get service/myquarkus -o jsonpath="{.spec.ports[*].nodePort}")
```

Comment:
this command will  mislead if we have two ports exposed by the app. Can we update this like `kubectl get svc myapp -ojsonpath="{.spec.ports[?(@.port==8080)].nodePort}"` ? We can also add reference to https://kubernetes.io/docs/reference/kubectl/cheatsheet/#interacting-with-running-pods